src/components/DisplayBlogs/DisplayBlogs.js

```diff
@@ -1,18 +1,39 @@
 import React, { useEffect, useState } from 'react';
 import {API, graphqlOperation} from 'aws-amplify';
-import {Card, Button, Dropdown, Modal} from 'react-bootstrap';
+import {Card, Button, Dropdown, Modal, InputGroup, FormControl} from 'react-bootstrap';
 import {Link, BrowserRouter as Router, Route} from 'react-router-dom'
 
 import {listBlogs} from '../../graphql/queries';
-import {onCreateBlog, onDeleteBlog} from '../../graphql/subscriptions'
+import {onCreateBlog, onDeleteBlog, onUpdateBlog} from '../../graphql/subscriptions'
 import style from './style.module.css'
 import CreateBlogs from '../CreateBlogs/CreateBlogs';
 import {HiDotsVertical} from 'react-icons/hi'
-import { deleteBlog } from '../../graphql/mutations';
+import { deleteBlog, updateBlog } from '../../graphql/mutations';
+import {AiOutlineEdit} from 'react-icons/ai';
+import EditForm from '../EditForm/EditForm';
 
 function DisplayBlogs(props) {
     const [blogs, setBlogs] = useState([]);
-    // const [modalShow, setModalShow] = useState(false);
+    const [show, setShow] = useState(false);
+
+    const [formData, setFormData] = useState({
```

Comment:
Instead of initialising to object -- you can instead give intial value as null.

Reply:
okay rajesh